packages/block-editor/src/components/media-replace-flow/index.js

```diff
@@ -43,6 +48,7 @@ const MediaReplaceFlow = ( {
 		return select( 'core/block-editor' ).getSettings().mediaUpload;
 	}, [] );
 	const editMediaButtonRef = createRef();
+	const errorNoticeID = uniqueId();
```

Comment:
I think it'd be good to use the `prefix` argument to uniqueId, just to be on the very safe side:
https://lodash.com/docs#uniqueId

I'm thinking of a situation where maybe a plugin decides to role their own integer notice ids.

Something like 'block-editor/media-replace-flow/error-notice/' might serve as a good prefix.

Reply:
yes, good idea!